glyphs-reader/src/font.rs

```diff
@@ -2584,7 +2586,11 @@ fn preprocess_unparsed_plist(s: &str) -> Cow<str> {
 impl Font {
     pub fn load(glyphs_file: &path::Path) -> Result<Font, Error> {
         let mut font = Self::load_raw(glyphs_file)?;
-        font.propagate_all_anchors();
+        if font.custom_parameters.propagate_anchors.is_none()
+            || font.custom_parameters.propagate_anchors == Some(true)
```

Comment:
or you could do this even more concisely, while preserving the same logic

```suggestion
        // propagate anchors by default unless explicitly set to false
        if font.custom_parameters.propagate_anchors.unwrap_or(true) {
```

it unwraps the Option to get the bool or returns true if there is None